packages/frontend/src/components/common/modal/DeleteActionModal.tsx

```diff
@@ -26,11 +68,6 @@ const DeleteActionModal: FC<DeleteActionModalProps> = ({
     const { projectUuid } = useParams<{ projectUuid: string }>();
     const { mutate: deleteDashboard, isLoading: isDeleting } =
         useDeleteDashboardMutation();
-    const { data: relatedDashboards } = useDashboardsContainingChart(
-        projectUuid,
-        uuid,
```

Comment:
This hook was being called when deleting a dashboard. So I moved all of this to a separate component.